server/methods/core/orders.js

```diff
@@ -5,15 +5,96 @@ import accounting from "accounting-js";
 import { check } from "meteor/check";
+import { Streamy } from "meteor/yuukan:streamy";
 import { getSlug } from "/lib/api";
 import { Cart, Media, Orders, Products, Shops } from "/lib/collections";
 import * as Schemas from "/lib/collections/schemas";
 import { Logger, Reaction } from "/server/api";
+import nodemailer from "nodemailer";
+import Nexmo from "nexmo";
 
 /**
  * Reaction Order Methods
  */
 Meteor.methods({
+
+    /**
+   * orders/sendMail
+   * @summary notifies seller and user of order's movement via mail
+   * @param {Object} payload - details of the order
+   * @returns {void} returns workflow update result
+   */
+  "orders/sendMail": function (payload) {
+    check(payload, Object);
+
+    const mailOptions = {
+      from: '"Reaction Commerce" <wesumeh@gmail.com>',
+      to: payload.dataForOrderEmail.order.email,
+      subject: "New Order Placed",
+      html: `<table style='border: 1px solid black; text-align:center;'><tr><thead><td colspan='2'
+        style='background: gray; color: white; font-family: tahoma; '><h1>Reaction</h1></td></thead></tr><tbody>
+        <tr><td> Name <td> ${payload.dataForOrderEmail.order.items[0].title} </td></tr>
+        <tr><td> Status </td><td> ${payload.message} </td></tr>
+        <tr><td> Quantity </td><td> ${payload.dataForOrderEmail.order.items[0].quantity} </td></tr>
+        <tr><td> Price </td><td> ${payload.dataForOrderEmail.billing.total} </td></tr>
+        <tr><td> Image </td><td> ${payload.dataForOrderEmail.order.items[0].placeholderImage} </td></tr>
+        <tr><td> Order Date </td><td> ${payload.dataForOrderEmail.orderDate} </td></tr>
+        <tr><td colspan='2'>
+          <a href='${payload.dataForOrderEmail.homepage}'>
+            Ⓒ ${payload.dataForOrderEmail.copyrightDate} Reaction
+          </a>
+        </td></tr>
+        </tbody></table>`
+    };
+
+    Meteor.call("nodemailerTransporter", mailOptions);
+  },
+
+  /**
+   * nodemailer method for sending mails
+   * @summary notifies seller and user of order's movement via mail
+   * @param {Object} payload - details of the order
+   * @returns {void}
+   */
+  "nodemailerTransporter": function (payload) {
+    check(payload, Object);
+    const transporter = nodemailer.createTransport({
+      host: "smtp.gmail.com",
+      port: 465,
+      secure: true,
+      auth: {
+        user: process.env.EMAIL,
+        pass: process.env.EMAIL_PASSWORD
+      }
+    });
+    transporter.sendMail(payload);
+  },
+
+  /**
+   * orders/sendText
+   * @summary notifies seller and user of order's movement via mail
+   * @param {Object} payload - details of the order
+   * @returns {void}
+   */
+  "orders/sendText": function (payload) {
+    check(payload, Object);
+    const nexmo = new Nexmo({
+      apiKey: process.env.NEXMO_APIKEY,
+      apiSecret: process.env.NEXMO_APISECRET
+    });
+ 
+    nexmo.message.sendSms(
+      "Reaction", +2349055483634, `${payload.message}. Reaction Commerce`,
```

Comment:
@Charpell remove phone number and add as Env var

Reply:
Okay, the change has now be made @emmyakin